server/app/services/applicant/ApplicantService.java

```diff
@@ -594,7 +594,15 @@ private void notifyProgramAdmins(
   }
 
   private CompletionStage<Void> maybeNotifyTiSubmitter(
-      String tiEmail, long applicantId, long applicationId, String programName) {
+      Optional<String> maybeTiEmail,
+      long applicantId,
+      long applicationId,
+      String programName,
+      Optional<StatusDefinitions.Status> status) {
+    if (!maybeTiEmail.isPresent()) {
```

Comment:
Personally not a big fan of having an Optional as a param just for the purpose of making the method a no-op if the param is empty. How about we keep this a String, change the method to `notifyTiSubmitter`, and don't call it if the email is empty?